cmd/axelard/cmd/root.go

```diff
@@ -201,7 +204,12 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		SetGenesisAuthCmd(app.DefaultNodeHome),
 	)
 
-	server.AddCommands(rootCmd, app.DefaultNodeHome, newApp, export(encodingConfig), crisis.AddModuleInitFlags)
+	starterFlags := func(startCmd *cobra.Command) {
+		crisis.AddModuleInitFlags(startCmd)
+		startCmd.Flags().String(wasmdirFlag, filepath.Join("$HOME", "."+app.Name, "wasm"), "path for the cosmwasm directory")
```

Comment:
default should be an empty string. the home might be overridden to something other than `~/.axelar/` (such as on testnet) which would require this to always be overridden as well. If the string is empty, we can dynamically set it later to the true value.

Reply:
done